src/components/ScoreBox/WordScoreList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import TotalScore from './TotalScore'
+const WordScoreList = (props) => {
+  const {words, scores, totalScore}  = props
+
+  const wordsList = words.map(function(word, index) {
+    return (<li key={word.toString()+index}>{word}</li>)
```

Comment:
yes. There will be warnings if you dont add a key.
https://facebook.github.io/react/docs/lists-and-keys.html